pkg/webhook/v1beta1/pod/inject_webhook_test.go

```diff
@@ -510,23 +496,6 @@ func TestGetMetricsCollectorArgs(t *testing.T) {
 				"-stop-rule", earlyStoppingRules[1],
 				"-s-earlystop", katibEarlyStopAddress,
 			},
-			name: "Trial with EarlyStopping rules",
-		},
-		{
-			trial: func() *trialsv1beta1.Trial {
-				trial := testTrial.DeepCopy()
-				trial.ObjectMeta.Labels[consts.LabelExperimentName] = "invalid-name"
-				return trial
-			}(),
-			mCSpec: common.MetricsCollectorSpec{
-				Collector: &common.CollectorSpec{
-					Kind: common.StdOutCollector,
-				},
-			},
-			earlyStoppingRules: earlyStoppingRules,
-			katibConfig:        configv1beta1.MetricsCollectorConfig{},
-			name:               "Trial with invalid Experiment label name. Suggestion is not created",
-			err:                true,
 		},
```

Comment:
What is the reason that we need to remove this case?

Reply:
Sorry,I might delete this test case by accident.